### PR TITLE
LPS-179001

### PR DESCRIPTION
**Pull request by marcosvictor22**
[LPS-110227 Automation](https://github.com/liferay-echo/liferay-portal/pull/14404/commits/22cfaf6d92c575779e6be786e66ebb6a30d29075)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by marcosvictor22** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 1]
```diff
@@ -2153,6 +2153,35 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-110227 Categories Navigation portlet can be seen within the Section where it was placed and can be opened/configured."
+	@priority = 5
+	test ViewTheNavigationPortletintheContentPaneloftheCurrentExperiencePage {
+		property portal.acceptance = "true";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:2159]: no need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 2]
```diff
@@ -2153,6 +2153,35 @@ definition {
...
+				title = "Web Content Title 1");
+
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = ${randomSiteName});
+		}
+
+		task ("Given that the user has created a Publisher portlet") {
+			PageEditor.addWidget(portletName = "Asset Publisher");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:2174]: Add Category Navigation portlet

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 3]
```diff
@@ -2153,6 +2153,35 @@ definition {
...
+			Button.clickPublish();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:2175]: No need to publish

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 4]
```diff
@@ -2153,6 +2153,35 @@ definition {
...
+		}
+
+		task ("Then it creates new experience for this page") {
+			PageEditor.addExperience(
+				experienceName = "New Experience",
+				segmentName = "Anyone");
+		}
+	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:2183]: Add steps
1. view Asset Publisher portlet and Category Navigation portlet.
2. Open the configuration modal of Category Navigation

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 5]
```diff
@@ -2153,6 +2153,35 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-110227 Categories Navigation portlet can be seen within the Section where it was placed and can be opened/configured."
+	@priority = 5
+	test ViewTheNavigationPortletintheContentPaneloftheCurrentExperiencePage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:2158]: Migrate to ContentPagesWithExperiencePersonalization.testcase then rename to ConfigureCategoryNavigationInNewExperience

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 6]
```diff
@@ -2153,6 +2153,35 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-110227 Categories Navigation portlet can be seen within the Section where it was placed and can be opened/configured."
+	@priority = 5
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:2157]: @priority = 4

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @marcosvictor22, could you update according to comments? Thanks

**Comment by Tim-Cao** [comment]
@marcosvictor22, see changes https://github.com/liferay-echo/liferay-portal/pull/14496
One point, we should use automation ticket number in our commit message.

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**